pkgs/os-specific/linux/prl-tools/default.nix

```
@@ -1,48 +1,55 @@
 { stdenv, lib, makeWrapper, p7zip
 , gawk, util-linux, xorg, glib, dbus-glib, zlib
 , kernel ? null, libsOnly ? false
-, undmg, fetchurl
+, fetchurl, undmg, perl, autoPatchelfHook
 }:
 
 assert (!libsOnly) -> kernel != null;
 
-let xorgFullVer = lib.getVersion xorg.xorgserver;
-    xorgVer = lib.versions.majorMinor xorgFullVer;
-    x64 = if stdenv.hostPlatform.system == "x86_64-linux" then true
-          else if stdenv.hostPlatform.system == "i686-linux" then false
-          else throw "Parallels Tools for Linux only support {x86-64,i686}-linux targets";
+let
+  aarch64 = stdenv.hostPlatform.system == "aarch64-linux";
+  x86_64 = stdenv.hostPlatform.system == "x86_64-linux";
+  i686 = stdenv.hostPlatform.system == "i686-linux";
+  _ = if (aarch64 || x86_64 || i686) then true
+      else throw "Parallels Tools for Linux only supports {aarch64,x86_64,i686}-linux targets";
 in
 stdenv.mkDerivation rec {
-  version = "${prl_major}.2.1-41615";
-  prl_major = "12";
+  version = "${prl_major}.1.1-51537";
+  prl_major = "17";
   pname = "prl-tools";
 
   # We download the full distribution to extract prl-tools-lin.iso from
   # => ${dmg}/Parallels\ Desktop.app/Contents/Resources/Tools/prl-tools-lin.iso
   src = fetchurl {
     url =  "https://download.parallels.com/desktop/v${prl_major}/${version}/ParallelsDesktop-${version}.dmg";
-    sha256 = "1jwzwif69qlhmfky9kigjaxpxfj0lyrl1iyrpqy4iwqvajdgbbym";
+    sha256 = "1ab5jwbg3jgvfwx8kwxwhhrsdp9wz4g9q7fg4z0hhd8v9pgy4yxx";
   };
 
   hardeningDisable = [ "pic" "format" ];
 
-  # also maybe python2 to generate xorg.conf
-  nativeBuildInputs = [ p7zip undmg ] ++ lib.optionals (!libsOnly) [ makeWrapper ] ++ kernel.moduleBuildDependencies;
+  nativeBuildInputs = [ p7zip undmg perl autoPatchelfHook ]
+    ++ lib.optionals (!libsOnly) [ makeWrapper ] ++ kernel.moduleBuildDependencies;
```

```suggestion
  nativeBuildInputs = [ p7zip undmg perl autoPatchelfHook ]
    ++ kernel.moduleBuildDependencies
    ++ lib.optionals (!libsOnly) [ makeWrapper ];
```
To make clear that ``kernel.moduleBuildDependencies`` is not conditional.